ui/app/styles/components/allocation-fs.scss

```diff
@@ -0,0 +1,70 @@
+.fs-explorer.table {
```

Comment:
I'm also wondering if the `.fs-explorer` and `.table` coupling is necessary. Sure it requires a `<table>` but does that table have to be a `.table`?

Reply:
agreed, unnecessary!